src/api.ts

```diff
@@ -109,6 +112,10 @@ export const apiFactory = (serviceId: number) => {
 			return res.sendStatus(400);
 		}
 
+		if (workerMap[req.body.common_name] !== req.params.worker) {
```

Comment:
Is it worth adding a sentry exception here so we can see if this happens more than expected? (at least in the short-term)

Reply:
Great idea, done!